lib/rubocop/cop/i18n/gettext/decorate_function_message.rb

```diff
@@ -22,14 +24,6 @@ def on_send(node)
             end
           end
 
-          def autocorrect(node)
-            if node.str_type?
-              single_string_correct(node)
-            elsif interpolation_offense?(node)
-              #              interpolation_correct(node)
```

Comment:
could we fix the autocorrect instead of removing ? 

I wish the Rails/i18n would also have auto correction